src/version-util.ts

```diff
@@ -18,8 +18,7 @@
  * Contains version utility functions.
  */
 
-import https from 'https';
-import { URL } from 'url';
+import * as httpm from '@actions/http-client';
```

Comment:
Can we import {HttpClient} instead? I know the tree-shaking will happen later, but it's nice to avoid extra imports where possible.

Reply:
Done